### PR TITLE
Ncgames 7

### PR DESCRIPTION
**Pull request by gracefaz**
#7 Complete: Added comment_count to /api/reviews/:review_id and edited previous test. Also added test to respond with empty array if no comments exist for specific review_id.

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -41,9 +41,8 @@ describe("GET /api/categories", () => {
   });
 });
 
-describe("GET /api/reviews/:review_id", () => {
-  test("200: responds with a single matching review", () => {
-    //const REVIEW_ID = 2;
```
Comment (hevdevs) [inline __tests__/app.test.js:46]: Remove commented out code before merging

File: __tests__/app.test.js [hunk 2]
```diff
@@ -75,7 +96,6 @@ describe("GET /api/reviews/:review_id", () => {
       .get("/api/reviews/99999")
       .expect(404)
       .then(({ body }) => {
-        //const { message } = response.body;
```
Comment (hevdevs) [inline __tests__/app.test.js:78]: here too

**Review by hevdevs — APPROVED** [review]
Looks good Grace, have you error tested for an endpoint of /api/reviews/notanid too?
